modules/how-huge-pages-are-consumed-by-apps.adoc

```diff
@@ -23,6 +23,7 @@ metadata:
   generateName: hugepages-volume-
 spec:
   containers:
+    # TODO: How to update this example since it already has securityContext set
```

Comment:
@s-urbaniak @stlaz Need help with this one too please

Reply:
already privileged, we can leave this one as-is